### DISCUSSION TITLE
Hackaton

### DISCUSSION THREAD

**Discussion by alexisgohe** [Launch-X-Latam/MisionFrontEnd#497] [Team Posts]
Que tal compañeros, estoy en busca de un equipo para el hackaton, cuento con conocimiento en HTML, CSS y un poco de JavaScript y React. Tambien no se me da tan mal el tema del UI entonces creo que puedo aportar buenas cosas. Quiero tomarme muy en serio el tema del hackaton por lo que si alguien está interesado en añadirme a su equipo o a formar uno nuevo, aquí les dejo mi correo de contacto: lx17035@innovaccion.mx :D

**Comment by VAlonzoGO** [comment]
animo amigo me uno lx05204@innovaccion.mx.
crea un grupo en teams y agregame, he enviado a algunos equipos pero aun no crean equipos en teams

**Comment by luisft14** [comment]
me uno a tu equipo mi correo es: lx04610@innovaccion.mx

**Comment by Reyesoy** [comment]
Hola, me gustaría formar parte de tu equipo, mi correo es  lx02278@innovaccion.mx, 
Espero me informen si me agregan
Saludos

**Comment by May-Alvarez** [comment]
Hola, me gustaria unirme mi correo es lx00319@innovaccion.mx
